sopel-modules/rat-board.py

```diff
@@ -910,6 +910,53 @@ def cmd_list(bot, trigger, params=''):
         -@: Show full case IDs.  (LONG)
 
     """
+    count = 0
+    plats = []
+    params = ['']
+    tmp = ''
+
+    for word in remainder:
```

Comment:
We may want to look into the parameterize wrapper to allow this and make this prettier. But this should work for now.

Reply:
I tried and couldn't find a way at the time to allow it based on two optional parameters. I might see a way to do it that way now though.

Reply:
If you got an idea to make it pretty, please tell me, as I have run out of ideas making it prettier with two sets of optional parameters